adapter/oam.go

```diff
@@ -85,7 +84,7 @@ func CreateComponents(scfg StaticCompConfig) error {
 	}
```

Comment:
@code-sleuth Where is the schema being assigned?

Reply:
https://github.com/meshery/meshery-adapter-library/blob/8f581b2369730d0a2589d068e5066fa09d8a7808/adapter/oam.go#L124